app/workers/set_tenant_id_worker.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class SetTenantIdWorker < ApplicationJob
+
+  def perform(provider)
+    provider.update_column(:master, false)
```

Comment:
This needs to be removed, `master` should stay null because of a unique index on the column. Just not to forget.